streaming/text/convert/enwiki/mds/make_train_parallel.py

```diff
@@ -1,28 +1,68 @@
+# Copyright 2022 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Invoke create_pretraining_data.py on shard ranges on all the cores."""
+
 import os
+from argparse import ArgumentParser, Namespace
+
+
+def parse_args() -> Namespace:
+    """Parse command-line arguments.
+
+    Returns:
+        Namespace: Command-line arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--in_pattern',
+                      type=str,
+                      default='/tmp/enwiki_preproc/results4/part-%05d-of-00500')
```

Comment:
Is the `default` will always exist? Wondering do we need `default` or can we make it mandatory ?

Reply:
Done